features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64LRUCache.kt

```diff
@@ -0,0 +1,135 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder.base64
+
+import android.content.ComponentCallbacks2
+import android.content.res.Configuration
+import android.graphics.drawable.Drawable
+import android.graphics.drawable.DrawableContainer
+import android.graphics.drawable.LayerDrawable
+import android.util.LruCache
+import android.view.View
+import androidx.annotation.VisibleForTesting
+
+@Suppress("MagicNumber")
+internal object Base64LRUCache : Cache, ComponentCallbacks2 {
```

Comment:
Indeed, seems like `ImageWireframe#base64` requires it to be a string. In this case let's return a `String` indeed (while it will create a copy of byte array, but probably we don't have a way to create a String pointing to existing byte array without allocating a copy, because `String` is immutable and our array can be mutable).

We could store things as `String`, to avoid this allocation, but then it will be 2x memory if Android stores internally strings as UTF-16 as desktop JVMs do.

Some people had the similar idea about string storage https://stackoverflow.com/questions/54393179/does-android-use-string-compacting-behind-the-scenes, but no reply.